service/tbc/tbc.go

```diff
@@ -894,11 +1012,13 @@ func (s *Server) handleAddr(_ context.Context, p *peer, msg *wire.MsgAddr) {
 	}
 
 	if err := s.pm.PeersInsert(peers); err != nil {
-		log.Errorf("Insert peers: %v", err)
+		return fmt.Errorf("Insert peers: %w", err)
```

Comment:
Error strings shouldn't start with capitals, per https://go.dev/wiki/CodeReviewComments#error-strings
```suggestion
		return fmt.Errorf("insert peers: %w", err)
```